cmd/kube-apiserver/app/server.go

```diff
@@ -246,56 +235,32 @@ func Run(s *options.APIServer) error {
 		glog.Fatalf("Invalid Authentication Config: %v", err)
 	}
 
-	authorizationModeNames := strings.Split(s.AuthorizationMode, ",")
-
-	modeEnabled := func(mode string) bool {
-		for _, m := range authorizationModeNames {
-			if m == mode {
-				return true
-			}
-		}
-		return false
+	privilegedLoopbackToken := uuid.NewRandom().String()
+	selfClientConfig, err := s.NewSelfClientConfig(privilegedLoopbackToken)
+	if err != nil {
+		glog.Fatalf("Failed to create clientset: %v", err)
 	}
+	client, err := s.NewSelfClient(privilegedLoopbackToken)
+	if err != nil {
+		glog.Errorf("Failed to create clientset: %v", err)
```

Comment:
> I know this is just a code move, but shouldn't this be fatal instead of error?

I'm not ready to do it yet because the wiring in `test-integration` doesn't allow this code to succeed (didn't before, doesn't now).  Once we fix `test-integration` to use a "normal" startup flow, yes.